components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/IMessageTrigger.cs

```diff
@@ -74,8 +74,11 @@ public interface IMessageTrigger : IDisposable {
         /// <summary>
         /// Run the group triggering
         /// </summary>
-        /// <param name="ct"></param>
-        /// <returns></returns>
         Task RunAsync(CancellationToken ct);
+
+        /// <summary>
+        /// Allow takeover of newly configured subscriptions
+        /// </summary>
+        void Reconfigure(object config);
```

Comment:
Why not get `WriterGroupJobModel` object directly here ?

Reply:
We're in the context of a generic assembly. We cannot take dependencies here on publisher related assembly.